site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1232,7 +1232,7 @@ private function amIBlindGrading($gradeable, $user, $peer) {
      *
      * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/grading/grade")
      */
-    public function showGrading($gradeable_id, $who_id = '', $from = "", $to = null, $gradeable_version = null, $sort = "id", $direction = "ASC", $to_ungraded = null, $component_id = "-1", $anon_mode = false) {
+    public function showGrading($gradeable_id, $who_id = '', $from = "", $to = null, $gradeable_version = null, $sort = "id", $direction = "ASC", $to_ungraded = null, $component_id = "-1", $anon_mode = false, $to_same_itempool = false) {
```

Comment:
Some of the lines are getting a little long here.  Would it be better to break some of the longer lines?